<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFeedback</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qfeedback.qrc">
    <normaloff>:/QFeedback/QFeedback.png</normaloff>:/QFeedback/QFeedback.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="messageTab">
       <attribute name="title">
        <string>Message</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="messagesSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="messageVLayout">
            <item>
             <layout class="QGridLayout" name="messageGridLayout">
              <item row="1" column="2">
               <widget class="QPushButton" name="messageRemoveButton">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="messageNewButton">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0" colspan="3">
               <widget class="QListWidget" name="messageList">
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="formLayoutWidget">
           <layout class="QFormLayout" name="messageFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="messageTypeLabel">
              <property name="text">
               <string>Type:</string>
              </property>
              <property name="buddy">
               <cstring>messageTypeSelect</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="messageTypeSelect">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="messageUrlLabel">
              <property name="text">
               <string>URL:</string>
              </property>
              <property name="buddy">
               <cstring>messageUrlEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="messageUrlEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="messageInfoLabel">
              <property name="text">
               <string>Extra info:</string>
              </property>
              <property name="buddy">
               <cstring>messageInfoEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="messageHLayout"/>
            </item>
            <item row="2" column="1">
             <widget class="QPlainTextEdit" name="messageInfoEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QGroupBox" name="messageExportGroup">
              <property name="title">
               <string>Export message</string>
              </property>
              <layout class="QFormLayout" name="formLayout_3">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="messageExportHeaderLabel">
                 <property name="text">
                  <string>Header:</string>
                 </property>
                 <property name="buddy">
                  <cstring>messageExportHeaderEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPlainTextEdit" name="messageExportHeaderEdit">
                 <property name="tabChangesFocus">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="messageExportFooterLabel">
                 <property name="text">
                  <string>Footer:</string>
                 </property>
                 <property name="buddy">
                  <cstring>messageExportFooterEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPlainTextEdit" name="messageExportFooterEdit">
                 <property name="tabChangesFocus">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="messageExportCopyButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Copy message</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="typeTab">
       <attribute name="title">
        <string>Types</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="typesSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <layout class="QGridLayout" name="typesGridLayout">
            <item row="0" column="0" colspan="3">
             <widget class="QListWidget" name="typesList">
              <property name="showDropIndicator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="typesNewButton">
              <property name="text">
               <string>New</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="typesRemoveButton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <spacer name="typesListButtonSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="formLayoutWidget_2">
           <layout class="QFormLayout" name="typesFormLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="typesNameLabel">
              <property name="text">
               <string>Name:</string>
              </property>
              <property name="buddy">
               <cstring>typesNameEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="typesNameEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="typesHeaderTextLabel">
              <property name="text">
               <string>Header text:</string>
              </property>
              <property name="buddy">
               <cstring>typesHeaderTextEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="typesExtraInfoLabel">
              <property name="text">
               <string>Extra info:</string>
              </property>
              <property name="buddy">
               <cstring>typesExtraInfoEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="typeHLayout">
              <item>
               <spacer name="typesSaveLeftSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="typesSaveButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="typesSaveRightSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QPlainTextEdit" name="typesHeaderTextEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPlainTextEdit" name="typesExtraInfoEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="tabChangesFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>messageList</tabstop>
  <tabstop>messageNewButton</tabstop>
  <tabstop>messageRemoveButton</tabstop>
  <tabstop>messageTypeSelect</tabstop>
  <tabstop>messageUrlEdit</tabstop>
  <tabstop>messageInfoEdit</tabstop>
  <tabstop>typesList</tabstop>
  <tabstop>typesNewButton</tabstop>
  <tabstop>typesRemoveButton</tabstop>
  <tabstop>typesNameEdit</tabstop>
  <tabstop>typesHeaderTextEdit</tabstop>
  <tabstop>typesExtraInfoEdit</tabstop>
  <tabstop>typesSaveButton</tabstop>
 </tabstops>
 <resources>
  <include location="qfeedback.qrc"/>
 </resources>
 <connections/>
</ui>
